Sources/PrimerSDK/Classes/User Interface/TokenizationViewModels/IPay88TokenizationViewModel.swift

```diff
@@ -0,0 +1,547 @@
+            ErrorHandler.handle(error: err)
+            errors.append(err)
+        }
+        
+        if PrimerAPIConfiguration.current?.clientSession?.customer?.mobileNumber == nil {
+            let err = PrimerError.invalidClientSessionValue(
+                name: "customer.mobileNumber",
+                value: nil,
+                userInfo: ["file": #file, "class": "\(Self.self)", "function": #function, "line": "\(#line)"],
+                diagnosticsId: nil)
+            ErrorHandler.handle(error: err)
+            errors.append(err)
+        }
+        
+#if !canImport(PrimerIPay88SDK)
+        let err = PrimerError.missingSDK(
+            sdkName: "PrimerIPay88SDK",
+            userInfo: ["file": #file, "class": "\(Self.self)", "function": #function, "line": "\(#line)"],
+            diagnosticsId: nil)
+        ErrorHandler.handle(error: err)
+        errors.append(err)
+#endif
+        
+        if errors.count == 1 {
+            throw errors.first!
+            
+        } else if errors.count > 1 {
+            let err = PrimerError.underlyingErrors(
+                errors: errors,
+                userInfo: ["file": #file, "class": "\(Self.self)", "function": #function, "line": "\(#line)"],
+                diagnosticsId: nil)
+            ErrorHandler.handle(error: err)
+            throw err
+        }
+    }
+    
+    override func performPreTokenizationSteps() -> Promise<Void> {
+        let event = Analytics.Event(
+            eventType: .ui,
+            properties: UIEventProperties(
+                action: .click,
+                context: Analytics.Event.Property.Context(
+                    issuerId: nil,
+                    paymentMethodType: self.config.type,
+                    url: nil),
+                extra: nil,
+                objectType: .button,
+                objectId: .select,
+                objectClass: "\(Self.self)",
+                place: .paymentMethodPopup))
+        Analytics.Service.record(event: event)
+        
+        PrimerUIManager.primerRootViewController?.showLoadingScreenIfNeeded(imageView: self.uiModule.makeIconImageView(withDimension: 24.0), message: nil)
+        
+        return Promise { seal in
+#if canImport(PrimerIPay88SDK)
+            firstly {
+                self.validateReturningPromise()
+            }
+            .then { () -> Promise<Void> in
+                let clientSessionActionsModule: ClientSessionActionsProtocol = ClientSessionActionsModule()
+                return clientSessionActionsModule.selectPaymentMethodIfNeeded(self.config.type, cardNetwork: nil)
+            }
+            .then { () -> Promise<Void> in
+                return self.handlePrimerWillCreatePaymentEvent(PrimerPaymentMethodData(type: self.config.type))
+            }
+            .done {
+                seal.fulfill()
+            }
+            .ensure {
+
+            }
+            .catch { err in
+                seal.reject(err)
+            }
+            
+#else
```

Comment:
Same thing here: we shouldn't return `iPay88` in the list of payment methods if the SDK is missing

Reply:
`iPay88` would never be here, because the view model will be removed from the `PrimerAPIConfiguration.paymentMethodConfigViewModels` if `PrimerIPay88SDK` is missing.